### ISSUE TITLE
Logging with origami.

### ISSUE THREAD

**Issue by fristonio** [Cloud-CV/origami-lib#38]
I was thinking that we can provide origami-lib with a logger functionality which will set up the logger for the demo. So a single method will bootstrap the python `logging` library which can then be used by demo user.

This will help us when we are deploying demos to our server too, suppose user want to check the logs for his demo the logger will store them in a file which can then be provided to demo owner. 

Thoughts on this?
@deshraj @uttu357 @AvaisP @dexter1691  @vpn1997

**Comment by uttu357** [comment]
@fristonio Are you thinking of adding something on the lines of https://github.com/Cloud-CV/origami-daemon/pull/4?

**Comment by fristonio** [comment]
Yeah, mostly like this. This way we can log better logs without needing the user to configure anything.

**Comment by uttu357** [comment]
Yeah sounds good. Also origami-lib should send logs with a certain label to the Origami web application for display to the user of the demo.

**Comment by fristonio** [comment]
Yeah, I have it in the todo list for origami-lib, but the thing is I didn't wanna mangle too much with the code that's already there in the frontend. We can do this easily once we have a basic working app with us.

**Comment by uttu357** [comment]
Sounds good! :+1:
